### ISSUE TITLE
How to migrate to local

### ISSUE THREAD

**Issue by nytram-md** [greghesp/bin-collection-api#13]
Hi

Sorry to hear that this venture didnt work out but you got me out of a hole a couple of months ago with this.

How do we go about storing this ourselves for personal links ?

Never done any of this stuff before.

Id like to keep local and call once per day. My Council was been scraped by your code.

Thanks

Martyn

**Comment by wifispray** [comment]
I have setup another online service, if you want to test out?

**Comment by nytram-md** [comment]
Definitely mate. I will try. Just send me the information and I'll do when I get home 👍

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by wifispray** [comment]
Just add your UPRN and your away.... https://binapi.minsky-home.co.uk/?uprn=

**Comment by wifispray** [comment]
BTW, first time will be slow, but there is local caching so it will be quicker onwards :)

**Comment by nytram-md** [comment]
Just tried and it pulled the information very fast but looks like it pulls the wrong day of the week. Our collection day is Tuesday and its pulling a Wednesday.

**Comment by nytram-md** [comment]
![image](https://user-images.githubusercontent.com/55548608/160199435-3e527d09-22ee-4a7d-972a-d0f5a0aeb96d.png)

Its always been Tuesday and nothing changed. Do you want my uprn ?

Thanks again

Martyn

**Comment by wifispray** [comment]
Yeah. Send it on over

**Comment by nytram-md** [comment]
UPRN is 63198222

Thanks

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by nytram-md** [comment]
I can see you have managed to get it to show the correct day now 👍 but is there anyway to include the garden bin now as it's only pulling the household waste and mixed recycling still ?

Thanks again

Cheers

Martyn

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by nytram-md** [comment]
Hi

Fantastic you got the date sorted. Garden bin is missing, is there any way to get that on also ?

Also, on collectiondate, it says null and collection day is invalid date. Is there a way to get it to say today ?

Thanks

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by nytram-md** [comment]
Hi

Any chance of Garden Bin been included in this ?

Thanks

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by wifispray** [comment]
I’m trying. Currently the data shows as blank

**Comment by wifispray** [comment]
For example if you check now.....

{"bins":[{"binType":"Household waste","collectionDate":"2022-04-14T00:00:00.000Z","collectionDay":"Thursday"},{"binType":"Mixed recycling","collectionDate":"2022-04-07T12:10:08.128Z","collectionDay":"Thursday"},{"binType":"Garden waste recycling","collectionDate":null,"collectionDay":"Invalid Date"}]}

**Comment by nytram-md** [comment]
Yeah, I've checked that URL and I agree it's only showing 2 bins but the actual council site shows 3 bins.

Never been an issue as the garden bin was not available when the first attempt Greg started.

Thanks

Martyn

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by wifispray** [comment]
Try now?

**Comment by nytram-md** [comment]
Fantastic, working. Thank you very much 👍

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by nytram-md** [comment]
> Try now?

Hi

Noticed the bins didn't update today. 

Checked the URL and getting a 502 Bad Gateway on your server.

Thanks

Martyn

**Comment by nytram-md** [comment]
If it's easier for me to run on my server locally for myself. Then if you can show me how. I will do.

👍
